ground/gcs/src/plugins/config/config.pro

```diff
@@ -47,7 +48,8 @@ HEADERS += calibration.h \
     configmodulewidget.h \
     configosdwidget.h \
     convertmwrate.h \
-    expocurve.h
+    expocurve.h \
+    autotuneshareform.h
```

Comment:
Config is getting into a massive library with no separation between logic and UI which is a shame.